docs/config-api.md

```diff
@@ -175,6 +175,13 @@ System.config({
 });
 ```
 
+* `format`: Overrides in what format the module is loaded. See [Module Formats](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md).
+* `exports`: Used to create a global variable, pointing to this meta path. Only available at the time of executing the module referenced by meta path. When meta is used inside `packages` configuration this variable is only available within that package. More info: [Exports](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md#exports)
```

Comment:
This is incorrect. "For loading a legacy global module format, sets the global value to be taken as the module exports".